ts/features/itwallet/common/saga/index.ts

```diff
@@ -9,6 +9,7 @@ import {
   checkWalletInstanceInconsistencySaga,
   checkWalletInstanceStateSaga
 } from "../../lifecycle/saga/checkWalletInstanceStateSaga";
+import { checkHasNfcFeatureSaga } from "../../identification/saga/index.ts";
```

Comment:
For consistency
```suggestion
import { checkHasNfcFeatureSaga } from "../../identification/saga";
```

Reply:
Applied in https://github.com/pagopa/io-app/pull/6985/commits/89c9138d0a7187ea6621bf24cfb0baf8afa9594a